testcases/open_posix_testsuite/conformance/interfaces/feof/1-1.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is licensed under the GPL license.  For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *
+ *	The feof function shall test the end-of-file indicator for
+ *	the stream pointed to by argument and the function shall return
+ *	non-zero if and only if the end-of-file indicator is set for stream
+ *
+ * method:
+ *	-open file in write mode
+ *	-write 2 sample strings into the file
+ *	-close the file
+ *	-open the same file in read mode
+ *	-read first string from file and check feof return value
+ *	-read second string from file and check feof return value
+ */
+
+#define _XOPEN_SOURCE 700
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include "posixtest.h"
+
+#define TNAME "feof/1-1.c"
```

Comment:
These test must not touch files outside of /tmp/.

Reply:
Updated to create a file inside /tmp

Reply:
I would actually go a step further and use a temporary file, provided by `tmpfile()`/`tmpnam(..)`: http://pubs.opengroup.org/onlinepubs/009604599/functions/tmpfile.html .

Using a temporary file function like `tmpfile()`/`tmpnam(..)` would achieve the following:
1. Honor `$TMPDIR`, which ensures that the end-user provided path (defaults to /tmp) is used instead of a hardcoded temporary directory.
1. Generate a unique `FILE*` handle or filename (at a point in time), reducing the potential for unwanted determinism issues when running the test multiple times.

Reply:
Updated the file creation process using tmpnam().